Update synchronizing-with-effects.md (#6856)
reactjs/react.dev: src/content/learn/synchronizing-with-effects.md

To write an Effect, follow these three steps:

> 1. **Declare an Effect.** By default, your Effect will run after [commit phase](/learn/render-and-commit) i.e. when changes are comitted to the dom.

Reviewer: ```suggestion
1. **Declare an Effect.** By default, your Effect will run after every [commit](/learn/render-and-commit).
```
